backend/drive/drive.go

```diff
@@ -229,8 +229,8 @@ a non root folder as its starting point.
 			Hide:     fs.OptionHideConfigurator,
 			Advanced: true,
 		}, {
-			Name:     "team_drive",
```

Comment:
As @darthShadow pointed out this will break people's configs which we try very hard not to do.

So if we want to do this, we need to read the old key name also.